contracts/staking/NativeTokenStakingManager.sol

```diff
@@ -27,7 +27,7 @@ contract NativeTokenStakingManager is
     }
 
     // solhint-disable ordering
-    function initialize(PoSValidatorManagerSettings calldata settings) external initializer {
+    function initialize(PoSValidatorManagerSettings calldata settings) external reinitializer(2) {
```

Comment:
If a PoA contract is upgraded separate from PoA->PoS migration, will this still work when we go to do the PoA->PoS migration?

Reply:
depends how the upgrade is done for the PoA before, but yes in most cases if it's just changing some of the logic of  the original contract.